### DISCUSSION TITLE
range 5%-95%

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1379] [Questions & Problems]
Hello everyone, I am recently using PK-sim for DDI modeling, and I want to obtain a 90% confidence interval for the simulated AUC. Is this available in the software? Or how can we caulate it? Looking forward to your answers!
![5%-95%-1](https://user-images.githubusercontent.com/95362728/201447980-1c783db3-e953-470d-b7ff-5140552e8db4.jpg)
![5%-95%-2](https://user-images.githubusercontent.com/95362728/201447990-832339d6-74fe-494f-b294-b7e931eda3dd.jpg)

**Comment by StephanSchaller** [comment]
Dear @WTY-tt , for such an output you would need to run a population simulation. In V11.1, this will be then available as an output. For now, you can export the individual values of AUCR and calculate the 90-10 CI in excel.

**Reply by WTY-tt** [reply]
Thank you for your reply. I'm currently experimenting with this approach while looking forward to the release of a new version, which I received a reply yesterday, and will be releasing a new version with this feature in the near future.
